src/__tests__/helpers_test.js

```diff
@@ -30,7 +30,15 @@ describe('helpers', () => {
     });
     it('should create an asyncOperation key with multiple params', () => {
       const asyncOperationKey = generateAsyncOperationKey('UPDATE_PERSON_DATA', { orgId: 222, personId: 111 });
-      expect(asyncOperationKey).to.equal('UPDATE_PERSON_DATA_222_111');
+      expect(asyncOperationKey).to.equal('UPDATE_PERSON_DATA_111_222');
```

Comment:
What happens when you have:
`{ orgId: 111, personId: 222 }` and `{ orgId: 222, personId: 111 }`?

Reply:
I just added tests to check! Looks like it still works.